src/components/CreateVmWizard/helpers.js

```diff
@@ -151,11 +151,21 @@ const isOsLinux = (operatingSystemId, operatingSystems) => {
   return os && !os.get('isWindows')
 }
 
+const verifyOtherOsId = (selectedOs, clusterId, { clusters, operatingSystems }) => {
+  if (clusterId && [ '', '0', undefined, '_' ].includes(selectedOs)) {
+    const osList = createOsList(clusterId, clusters, operatingSystems)
+    const otherOs = osList.find(os => os.value === 'Other OS')
```

Comment:
`Other OS` is just English label - check [osinfo-defaults.properties](https://github.com/oVirt/ovirt-engine/blob/2314e3b31e42ff1c96395948af7a9a7f996f56ab/packaging/conf/osinfo-defaults.properties#L38) 
> name is I18N if the one creates a 01-os_${LOCALE}.properties with the co-responding name property

Note also that some OS of type `other` have different English labels i.e.
> os.other_linux.name.value = Linux

I think we should base our logic on unique OS names  and/or IDs.